src/os/data/columndefinition.js

```diff
@@ -27,9 +26,9 @@ os.data.ColumnDefinition = function(opt_name, opt_field) {
 
   /**
    * The field
-   * @type {string}
+   * @type {!string}
```

Comment:
`!` is implied for primitives. You also removed the `|| ''`, which would allow this value to be null/undefined if `opt_name` and `opt_field` aren't provided.